test/integration/test_timeout.py

```diff
@@ -0,0 +1,224 @@
+#  © Copyright IBM Corporation 2020.
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#       http://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import time
+import unittest
+
+import requests
+from unittest.mock import Mock
+from requests import Response
+from urllib3 import Timeout
+
+from ibmcloudant.cloudant_v1 import CloudantV1
+from ibm_cloud_sdk_core.authenticators import BasicAuthenticator, IAMAuthenticator, NoAuthAuthenticator
+from ibmcloudant.couchdb_session_authenticator import CouchDbSessionAuthenticator
+
+DEFAULT_TIMEOUT = 150  # 2.5m (=150s)
+CUSTOM_TIMEOUT = 10    # 10s
+CUSTOM_TIMEOUT_CONFIG = {'timeout': CUSTOM_TIMEOUT}
+
+class Helpers():
+
+    @staticmethod
+    def get_current_time_plus_two_minute() -> int:
+        return int(time.time()) + 120
+
+    @staticmethod
+    def get_mocked_response():
```

Comment:
While it is perfectly functional for the purposes of the test it is a little confusing to use the same mock response for both auth and a service requests. Maybe at least put a comment explaining that.

Reply:
I've added some description: 24bad01 WDYT?